Gemfile

```diff
@@ -15,11 +15,21 @@ gem 'jquery-rails'
 gem 'turbolinks', '~> 5.x'
 gem 'jbuilder', '~> 2.0'
 
+# User management
+gem 'devise'
+gem 'cancancan'
+
+# Backend
+gemspec path: 'backend'
+
 # Templating
 gem 'slim-rails'
```

Comment:
Changing the templating system (**slim**) and using a different form helpers (**simple_form**), is something that I find to be counter-productive, and which I think should not be done by default in our projects. Most of our developers are front end developers who are much more used to plain HTML, and already need to learn some rails to navigate the rails views. Adding extra abstractions on top of it just makes it more complex, removes the standards.

I'd prefer if these decisions were made as a team on a per project basis, instead of being taken as defaults.

For the moment let's keep it, but I'd suggest removing it later.